airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml

```diff
@@ -32,6 +32,14 @@ definitions:
     http_method: GET
     request_parameters: {}
     request_headers: {}
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - http_codes: [499]
+              action: FAIL
+              error_message: "Source Typeform was waiting to long time for response from Typeform API, please retry later."
```

Comment:
I would rephrase to something like : "Could not complete the stream: Source Typeform has been waiting for too long for a response from Typeform API. Please try again later."